bin/test-unit.rb

```diff
@@ -0,0 +1,9 @@
+base_dir = File.expand_path(File.join(File.dirname(__FILE__), ".."))
+lib_dir  = File.join(base_dir, "lib")
+test_dir = File.join(base_dir, "test")
+
+$LOAD_PATH.unshift(lib_dir)
+
+require 'test/unit'
+
+exit Test::Unit::AutoRunner.run(true, test_dir)
```

Comment:
copy and paste codes in reference
https://test-unit.github.io/test-unit/en/file.how-to.html